pkgs/tools/misc/entr/default.nix

```diff
@@ -28,5 +28,6 @@ stdenv.mkDerivation rec {
     license = licenses.isc;
     platforms = platforms.all;
     maintainers = with maintainers; [ pSub synthetica ];
+    changelog = "https://raw.githubusercontent.com/eradman/entr/4.7/NEWS";
```

Comment:
```suggestion
```
this is already defined.

Reply:
Sorry, I have accumulated these over time and it must have been defined since then. I have removed it.